throws句は、その後に記述される例外が発生したときに行う処理をあらわしている。